### ISSUE TITLE
Restore source pane scroll logic not working in latest Orion 1.0 build

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#3783]
@jhpedemonte 

On Mac/FF:

I put the following logic into Editor.js to restore the current source editor scroll amount:

```
                    var leftPixel = textView.getHorizontalPixel();
                    var topPixel = textView.getTopPixel();
                    that.handleChange(that._textModel.getText());
                    delete that.isTyping;
                    textView.setSelection(sel.start, sel.end, false); // show=false because we are handling scrolling ourselves
                    textView.setHorizontalPixel(leftPixel);
                    textView.setTopPixel(topPixel);

```

but whenever I make a source view change, the source pane jumps to the top (after the 1 second wait to see if more characters are being typed), just like it did before I added the above code for leftPixel and topPixel.

Is there a new Orion source editor in the Orion 1.0 build? Has something changed under the hood such that handleChange() is now asynchronous?

To reproduce:
- New desktop app
- Add a DIV, and inside the DIV add a Button and TextBox
- Split view (horizontal, so design is on top and source is on bottom)
- In source pane, scroll down to bottom and change the button label and wait a second

Result=> source pane jumps to top/left, cursor goes to first position

**Comment by JonFerraiolo** [comment]
Also happens run running development builds from Eclipse. Same thing happens with Mac/Chrome.

**Comment by peller** [comment]
fixed by reverting to 0.5 code in 84959c3b8fabb9faad4111d79fbbf8f06ff974ef
